projects/1.16/assets/draconic-evolution/draconicevolution/lang/zh_cn.json

```
@@ -288,306 +297,311 @@
   "module.draconicevolution.proj_grav_comp.value": "%s%%",
-  "module.draconicevolution.proj_penetration.info": "提高弹射物穿过目标并击中一个额外目标的几率",
-  "module.draconicevolution.proj_penetration.info2": "每穿过一个目标后减少25%的几率",
-  "module.draconicevolution.proj_penetration.name": "破防几率",
+  "module.draconicevolution.proj_penetration.info": "增加弹射物穿过目标并击中额外目标的几率。",
+  "module.draconicevolution.proj_penetration.info2": "每击中一个目标后降低25%。",
+  "module.draconicevolution.proj_penetration.name": "穿透几率",
   "module.draconicevolution.proj_penetration.value": "%s%%",
   "module.draconicevolution.proj_velocity.name": "速度",
-  "module.draconicevolution.proj_velocity.value": "%s%%（约%sm/s）",
-  "module.draconicevolution.shield_capacity.name": "可充能护盾点数",
+  "module.draconicevolution.proj_velocity.value": "%s%% (~%s m/s)",
+  "module.draconicevolution.shield_capacity.name": "护盾容量",
   "module.draconicevolution.shield_capacity.value": "%s点",
-  "module.draconicevolution.shield_control.name": "护盾冷却时间",
+  "module.draconicevolution.shield_control.name": "护盾冷却",
   "module.draconicevolution.shield_control.value": "%s秒",
-  "module.draconicevolution.shield_passive.name": "操作护盾消耗",
-  "module.draconicevolution.shield_passive.value": "%s OP/t",
-  "module.draconicevolution.shield_recharge.name": "护盾重充能",
-  "module.draconicevolution.shield_recharge.value": "%s 点每秒（共%s秒）",
+  "module.draconicevolution.shield_passive.name": "能量运行消耗",
+  "module.draconicevolution.shield_passive.value": "%sOP/t",
+  "module.draconicevolution.shield_recharge.name": "护盾恢复",
+  "module.draconicevolution.shield_recharge.value": "%s点/秒（%s秒\n消耗%sOP/t",
   "module.draconicevolution.speed.name": "速度",
   "module.draconicevolution.speed.value": "+%s%%",
   "module.draconicevolution.undying.charge.name": "充能时间",
-  "module.draconicevolution.undying.charge.value": "%s Seconds",
-  "module.draconicevolution.undying.energy.name": "Charge Energy",
+  "module.draconicevolution.undying.charge.value": "%s 秒",
+  "module.draconicevolution.undying.energy.name": "充能",
   "module.draconicevolution.undying.energy.value": "%sOP @%s OP/t",
-  "module.draconicevolution.undying.health.name": "Health Boost",
-  "module.draconicevolution.undying.health.value": "%s Health points",
-  "module.draconicevolution.undying.invuln.active": "Invulnerable for %s Seconds",
-  "module.draconicevolution.undying.invuln.name": "Invulnerable Time",
-  "module.draconicevolution.undying.invuln.value": "%s Seconds",
-  "module.draconicevolution.undying.shield.name": "Shield Boost",
-  "module.draconicevolution.undying.shield.value": "%s for %s seconds",
+  "module.draconicevolution.undying.health.name": "生命提升",
+  "module.draconicevolution.undying.health.value": "%s点生命值",
+  "module.draconicevolution.undying.invuln.active": "无敌%s秒",
+  "module.draconicevolution.undying.invuln.name": "无敌时间",
+  "module.draconicevolution.undying.invuln.value": "%s秒",
+  "module.draconicevolution.undying.shield.name": "护盾瞬间提升",
```

```suggestion
  "module.draconicevolution.undying.shield.name": "应急护盾充能",
```
主要是“瞬间”放在这里听上去有点口语化，如果我理解的没错的话是触发效果之后“迅速的为玩家额外充能xx点护盾值”，那么应该可以理解为“应急护盾”